tests/unit/schema/test.action.js

```diff
@@ -0,0 +1,62 @@
+import cloneDeep from 'lodash.clonedeep';
+
+import { validateAddon } from 'schema/validator';
+
+import { validManifest } from './helpers';
+import { assertHasMatchingError } from '../helpers';
+
+describe('/browser_action', () => {
```

Comment:
(I suppose we don't do this in this project, I like to use a top-level `describe(__filename, () => {});` in test files).

Reply:
yeah, I don't recall that to be used anywhere in this project at the moment.

isn't that redundant? from what I recall the filename would still be reported in the failures output and so at that point wouldn't be mostly the same to don't group the tests into a describe if there is only one and it is named exactly as the file?

Anyway, besides my doubt described above (which is not really a big deal), I don't really mind to use that convention in this new test files if we want to start using it in this project, but at that point I would split this test file in two separate ones, one for `action` and the other for `browserAction`.

Reply:
(I haven't done this, I think we can do it in a follow up if we think it would be valuable, even just to match conventions used in other ones of our projects).